shims/spark312/src/main/scala/com/nvidia/spark/rapids/shims/spark312/ParquetCachedBatchSerializer.scala

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2021, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.nvidia.spark.rapids.shims.spark312
+
+/**
+ * This is a wrapper to com.nvidia.spark.rapids.shims.spark311.ParquetCachedBatchSerializer
+ */
+class ParquetCachedBatchSerializer extends
```

Comment:
we should update our docs -> additional-functionality/cache-serializer.md. to clarify to use the correct version that matches spark version.